src/topology/metric_space/basic.lean

```diff
@@ -1561,13 +1561,20 @@ lemma bounded_bUnion {I : set β} {s : β → set α} (H : finite I) :
 finite.induction_on H (by simp) $ λ x I _ _ IH,
 by simp [or_imp_distrib, forall_and_distrib, IH]
 
-/-- A compact set is bounded -/
-lemma bounded_of_compact {s : set α} (h : is_compact s) : bounded s :=
--- We cover the compact set by finitely many balls of radius 1,
+/-- A totally bounded set is bounded -/
+lemma bounded_of_totally_bounded {s : set α} (h : totally_bounded s) : bounded s :=
```

Comment:
Can you maybe drop this name and only use the one with dot notation? Same thing for the next lemma.